### PR TITLE
Verification of page 1

### PR DESCRIPTION
**Pull request by nathanbrown823**

### CODE REVIEW

**Comment by whanley** [comment]
Okay, this is a start. Please thoroughly verify 25 divs.

File: 1907-11-05.xml [hunk 1]
```diff
@@ -1340,13 +1344,12 @@
...
-                    <p>SPA </p>
                     <p>A correspondent writes from Cairo - </p>
                     <p>The warfare between the parties continue, but what it all means Heaven, and
-                        perhaps, Sir Eldon Gorst only know. It is 30 fatally easy to create parties
-                        with the pen and then to discover that they have po exist </p>
+                        perhaps, Sir Eldon Gorst only know. It is so fatally easy to create parties
+                        with the pen and then to discover that they have no exist </p>
                     <p>ence save on paper. However no one is chary in of describing what are at most
-                        parties in poudre r as parties in win, and no one is any the worse </p>
+                        parties in posse as parties in esse, and no one is any the worse </p>
```
Comment (whanley) [inline 1907-11-05.xml:1352]: this doesn't make sense?

File: 1907-11-05.xml [hunk 2]
```diff
@@ -1305,10 +1309,10 @@
                 </div>
                 <div type="item">
                     <head>VANISHING CAIRO.</head>
-                    <p>The oil paintings and water colours of "V nishing Cairo" now being exhibited
-                        at the Dore Gallery, London, by Mr. R. M. Chevalier are the results of
+                    <p>The oil paintings and water-colours of "Vanishing Cairo" now being exhibited
+                        at the Doré Gallery, London, by Mr. R. M. Chevalier are the results of
                         prolonged study in Egypt So far back as the time of Arabi Pasha's attempt at
-                        military despotisin Mt, Chevalier was in the country, and in the comse of
+                        military despotisin Mr, Chevalier was in the country, and in the comnise of
```
Comment (whanley) [inline 1907-11-05.xml:1315]: unclear

File: 1907-11-05.xml [hunk 3]
```diff
@@ -1248,7 +1252,7 @@
                     <p>ot </p>
```
Comment (whanley) [inline 1907-11-05.xml:1252]: don't leave nonsense paragraphs

**Review by whanley — COMMENTED** [review]
You've made a few more changes to verification, but lots more to clean up, and some of the changes don't make sense. Remember, the aim is to make the xml match the image.

How about deliverable 1?

**Comment by whanley** [comment]
You are making progress! Good! The OCR that you've added for the 10-28 issue is great. It needs to be divided into divs, but that's doable. 

There are two problems to address in the short term.

First, you've put the content for 10-28 in the 1904-11-28 file. Copy everything you have there and put it in the 1904-10-28 file.

Second, your page and div hierarchy is a bit messed up. The problem is that you are not closing lots of your divs with `</div>` when they need to be closed. These instructions might help: https://dig-eg-gaz.github.io/how-to/digitization/polished-tei-instructions/#1-get-the-page-hierarchy-right

Keep at it!

**Comment by whanley** [comment]
I've added these files manually to the master content repository. You'll want to sync that repository before doing your next set of work.

**Comment by nathanbrown823** [comment]
Dear Professor Hanley,
Sorry I just saw this email chain. The fact that it was sent through GitHub made it go to my spam.
Sincerely,
-Nathan Brown

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request closed without merging**